src/header/Header.tsx

```diff
@@ -47,7 +47,9 @@ export const Topbar = () => (
     <div className={styles.topbarLeft}>
       <HomeLink />
       <div className={styles.logotypeWrapper}>
-        <Logotype className={styles.logotype} />
+        <Link to="/" aria-label="Ensembl home page">
+          <Logotype className={styles.logotype} />
+        </Link>
```

Comment:
<img width="379" height="35" alt="Image" src="https://github.com/user-attachments/assets/8cdd4ff1-a5e3-439d-8e15-eccc59c4a258" />

<img width="379" height="34" alt="Image" src="https://github.com/user-attachments/assets/3899dd13-054a-4bae-819d-82f0d2e18eb8" />

With keyboard navigation, it is now possible to focus the home button and the logo type independently, although they both navigate to the same place. Perhaps the logo type should not be focusable?